Shared/shared.template.json

```diff
@@ -180,6 +180,67 @@
       "metadata": {
         "description": "Properties for the elastic pool"
       }
+    },
+    "newStyleAppServicePlanPrefix": {
```

Comment:
Can we improve this name? Its not very descriptive of what or why its changed.

Reply:
Removed this parameter